packages/base-controller/tsconfig.build.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "../../tsconfig.packages.build.json",
+  "compilerOptions": {
+    "baseUrl": "./",
+    "outDir": "./dist",
+    "rootDir": "./src"
+  },
+  "include": ["../../types/**/*", "./src/**/*"]
```

Comment:
1. What's `../../types`?
2. The glob is unnecessary, you could do `["../../types", "src"]`.

Reply:
I've updated `include` to drop the `**/*` in the tsconfig files for each package.